mesonbuild/coredata.py

```diff
@@ -713,9 +713,11 @@ def replace(v):
 
         return dirty
 
-    def clear_deps_cache(self):
+    def clear_cache(self):
         self.deps.host.clear()
         self.deps.build.clear()
+        self.compiler_check_cache.clear()
+        self.run_check_cache.clear()
```

Comment:
It's a really good thing I'm removing the code that clears the cache whenever you set the wrap-mode, lol.

Reply:
In some of my WIP branch locally I already had a commit that adds `clear_cache()` that was calling `clear_deps_cache()`, but now we can just remove `clear_deps_cache()`. Note that this PR requires to merge yours first.